src/components/ui/card.tsx

```diff
@@ -0,0 +1,158 @@
+import { forwardRef, HTMLAttributes } from 'react';
+
+import { CalendarDays, CircleCheck } from 'lucide-react';
+
+import { cn } from '@/lib/utils';
+
+interface CardHeaderProps extends HTMLAttributes<HTMLDivElement> {
+  startDate: string;
+  endDate: string;
+}
+
+interface CardContentProps extends HTMLAttributes<HTMLDivElement> {
+  category: string;
+  tenant: string;
+  isComplete: boolean;
+}
+
+/**
+ * @example
+ * ```ts
+ * return (
+ * <Card>
+ *  <CardHeader startDate={startDate} endDate={endDate} />
+ *  <CardContent category={category} tenant={tenant} isComplete={isComplete} />
+ *  <CardFooter category={category} tenant={tenant} isComplete={isComplete} />
+ * </Card>
+ * )
+ * ```
+ */
+const Card = forwardRef<HTMLDivElement, HTMLAttributes<HTMLDivElement>>(
+  ({ className, ...props }, ref) => (
+    <div
+      ref={ref}
+      className={cn(
+        'rounded-xl bg-card text-card-foreground shadow-[0_4px_8px_0_rgb(0,0,0,0.25)]',
+        className,
+      )}
+      {...props}
+    />
+  ),
+);
+Card.displayName = 'Card';
+
+const CardHeader = forwardRef<HTMLDivElement, CardHeaderProps>(
+  ({ className, startDate, endDate, ...props }, ref) => (
+    <div
+      ref={ref}
+      className={cn(
+        'flex flex-col space-y-1.5  p-[14px] border-b-2 border-slate-300',
```

Comment:
Could you adjust the spacing between the `CardTitle` and `CardDescription`? It should be 8px, but yours is 6px.
<img width="189" alt="image" src="https://github.com/Tascurator/tascurator-frontend/assets/124953279/ff1cb58e-4a05-4ad2-abdc-090d7760cc28">